packages/desktop-electron/package.json

```diff
@@ -24,12 +24,6 @@
       "!node_modules/better-sqlite3/{benchmark,src,bin,docs,deps,build/Release/obj,build/Release/sqlite3.a,build/Release/test_extension.node}",
       "build"
     ],
-    "publish": {
```

Comment:
This is old - we used to publish to github for autoupdate but that hasn't worked in ages.